src/routes/CommunitySettings/TopicsSettingsTab/TopicsSettingsTab.scss

```diff
@@ -1,5 +1,11 @@
 .wrapper {
   height: 100%;
+
+  p {
+    color: #808C9B;
+    padding: 15px 0px;
+  }
+
 }
```

Comment:
NVM my PR Review comment. I see that this is the change that fixed the whitespace, I guess I just didn't really understand it and still don't but if it fixes it it fixes it :)

Reply:
Well I guess I do get it, it actually. Seems clear enough. We don't usually use element level rules in the CSS Module files. Why are we doing it here?

Reply:
well it was a mistake to to put it at the global level, that was the bug. but otherwise it can be useful